api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -52,8 +52,31 @@ message RateLimitRequest {
   uint32 hits_addend = 3;
 }
 
+// Cacheable quota for responses.
+// Quota can be granted at different levels: either for each individual descriptor or for the whole descriptor set.
+// This is a certain number of requests over a period of time.
+// The client may cache this result and apply the effective RateLimitResponse to future matching
+// requests containing a matching  without querying rate limit service.
+//
+// When quota expires due to timeout, a new RLS request will also be made.
+// The implementation may choose to preemptively query the rate limit server for more quota on or
+// before expiration or before the available quota runs out.
+// [#not-implemented-hide:]
+message Quota {
```

Comment:
Moving the `Quota` message out of `RateLimitResponse` seems like a breaking change.  I don't think that's okay.

Reply:
Thanks for catching this! I just added back the old `message Quota` without other changes based on understanding of https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md#message-level that the `protoxform` will take care of the annotation for me.

Reply:
I think protoxform is intended to fix problems where a message is renamed between major versions of the API; I don't think it helps in a case like this.

Why do you need to add this new copy of the `Quota` message to begin with?  The existing `Quota` message is within `RateLimitResponse`, and the new field you're adding is also within `RateLimitResponse`, so it looks like you should be able to use the existing copy as-is.  I don't think you need to add this new copy at all.

Reply:
I was thinking that moving it out will make it clearer of sharing the `Quota` message. But you're right they both belongs to `RateLimitResponse`.
Done.